activesupport/lib/active_support/multibyte/chars.rb

```diff
@@ -76,6 +76,11 @@ def respond_to_missing?(method, include_private)
       # Returns +true+ when the proxy class can handle the string. Returns
       # +false+ otherwise.
       def self.consumes?(string)
+        ActiveSupport::Deprecation.warn(<<-MSG.squish)
+          ActiveSupport::Multibyte::Chars#consumes? is deprecated and will be
+          removed from Rails 6.1. Use #is_utf8? instead.
```

Comment:
Let's indicate that usage changes from a String argument to a String method. Example:

> ActiveSupport::Multibyte::Chars.consumes?(string) is deprecated and will be removed from Rails 6.1. Use string.is_utf8? instead.